src/pykeen/pipeline/api.py

```diff
@@ -524,8 +524,8 @@ def replicate_pipeline_from_config(
 
 def _iterate_moved(pipeline_results: Iterable[PipelineResult]):
     for pipeline_result in pipeline_results:
-        pipeline_result.model.device = resolve_device("cpu")
-        pipeline_result.model.to_device_()
+        pipeline_result.model = pipeline_result.model.cpu()
```

Comment:
if this is in-place we can remove the reassignment

Reply:
There is some subtlety here: `cpu()` on modules is in-place, while `cpu()` on tensors is out-of-place. I added a comment there.

cf. also
* https://pytorch.org/docs/stable/generated/torch.Tensor.cpu.html
* https://pytorch.org/docs/stable/generated/torch.nn.Module.html